cmd/dexctl/main.go

```diff
@@ -69,19 +75,41 @@ func wrapRun(run func(cmd *cobra.Command, args []string) int) func(cmd *cobra.Co
 	}
 }
 
-func getDriver() (drv driver) {
+func isDBURLPresent() bool {
```

Comment:
Since this method is now being used very differently, it shouldn't be exiting on it's own. It should just return an error if not present.